notebooks/moran_matrix_example.ipynb

```diff
@@ -0,0 +1,323 @@
Ocj25G7DGVJ5QWhsailLam2j+T6QoODgxw9epQ9e/aU3RdaSsRVeuJU1vbGV72+bOMRhHwUe1I9CfwD8ArgQ0qpnwBfAr6ltU6U6fp3Ap8Evlhg//8Dzpn79xzgn+f+29DUS5eP4eFhDh06RDqd9t5LJBJordm0aVPFrpu7AqGUIhKJkEwmq97+0V8Esv1FL+Ws626o+DUFQagero21bTtL0EmlUszMzNDe3o5pGKw/fjSzQykePP95XJPz+ULnkTb1Qj1Q6/p81Yz6rZeFtXzMDD6dtb3xVW8o8ElBqE9qbUtK9ZHKVWTeFcAMw/AihctVv2g5EVcH/+rN3utyp5/We4F+oTYUE4VSWut7gXuVUmHgJcDvAp9SSn1Pa/17y7241vohpdT2Ih95KfBFnZFrf6qUWquU2qi1bvj8nnK2bFwKjuNw+PBhUqkUpmmilPJW2A4fPkxXV1fFDEW+AnLxeJyuri7Wr19fNSPlX81rvuAiEYQEYQXh7zTW09PDwMCAl/rl1hAKh8NYlsUlj3zPa8+y67IX4cTjJJNJIpGIdz7TNPOeR9rUC/WAP3pGa+29rlZaejUdOf+13PlTLRbW8nHko+/1XksdIaERqYcSF8V8pHKmtvkFMPfYcgpgSy2YnZV++rHPzxuzdHUTKkExUcj765iLDPoK8BWlVBvwm5Ue2BybgSO+7cG59xpeFFpul4/lkkwmmZmZwbIszxAopTBNk2QySTKZJBwOZx3T2tq67OsWW4E4duwYGzdurMrvMPCmV3uvN9zwe6y76kXzxikquiBUhpaWloqdO1+nse7ubs455xwOHjxIMpnEMAx27NhBLBbj4p/dh6M1oLm/awfq1ClCoVDeSW9vby8AsVjMO09/f7/3viDUikAggFKK6elpZmZmPEfKTf2oRvRMtRw5/7XC4TBaa9LpdFUX1vLhd+REEBIaFX8kniu4uoJQtSLxivlIw8PDZetM5hfA/PMSpZS3sJTrC0FpPsJSI67mpZ/O2cdy2FDp6iYUo5go9O/53tRaTwBfqMxw5pHvrzxvkqdS6lbg1rnN9RUbUZmpddpBriFxI4byceONNy77erVegXAch4E3vXou7FyTuv53menf6T3sREVf3eTYkdu11rfXcjwrlWuvvbZi53Y7hPmjeZ566in6+/u55pprvOgh0zTZ+u0vkJ6b+IIiEQgxOz5Ob29v3sgfpRR9fX10d3dnnUcQcqm2LTEMg1AoxMjIiOeoaK2ZmpqqaOSvn3I5cqU4XO613vzmN9fFAo7fkVt75QtrNg5h5VELW9LR0eGVl3Dnx5ZlsX379qreZ7k+UrlT2/wC2Atf+EK01kxPT5NIJHAch/3792f5AIvxEZbi78wcjmVt+9NPlyvo1DotUKh/CopCWuuPFNqnlDpba/10of1lZBDY6tveAgzl++CckbwdQCm1q/JDa2yampoIhUIkk0ksXxHEdDpdcJW8HNSy25idiPPUO16LbdsZ4/2K12G2tWcZVVHRVzd+OyI0Bv40MWDBDmGRSAStNQce30unbQOZFbivd52H5TisW7eO6enpebWD/Lht6gWhENW2JW76VEtLixcpBJmoPLfAabUm/Et15JayKFPrhTWAkw9kN+bdcMMrK3IdiWBendRqXpJOp5mdnS26WFxtlrOwnO/+yU17TSQSxONxIGM7c+sLLcZHWIq/c+Tj7/Neu9GGrm1frqBT60V5of4p2hJBKfVcMulaD2mtx5RSO4F3As8nW6ypFN8G/kQp9Z9kCkxPrIR6QvWAYRhs27bNWwlwjX4gEGDbtm15jct9990HwHXXXbes69aiyHby6GGe/vBfesaYP3o7BAIY4BnVDRs2iIouCFXgoYceAuDKK69c8LN+0ccv0uRLE9u8eTO2bRMMBkmn0xiG4dk29zyRSIRoNMrG7/wHWgEo7t32LIxUijVr1rB27VrieWoKCUI94074m5ubiUQipNNpLz08mUzWdMJfqjOSz+E6evQoqVSKLVu2LHteUilR5cS9d3uvK5E2JhHMQjVxa45qrQmFQt77tm1XvOboQixWaHEch9nZWU6cOMGxY8fy3j9uyunXvvY14vE4v/Irv0IkEvGe/0v1ERbr7+TWEfLf9+l0mqmpKVpaWrLmJYsRdGq5KC80BgVFIaXUh4FfB3YD71BK3Qu8AXg/8JpyXFwp9SXgamC9UmoQeC8QANBafwb4DvBi4AAQB24px3WFDBs3bkQpxejoqLcq7hrKfJSrFWW1i2xPPPxDxu52Mx41vOEvsva7RtUNF11NKrqsPAq14Pjx4/PeyxV/8ok+PT099Pb2opTKmyZ28OBBJicnOXnyJIDXWdEVkwYHB+np6WHTNz8PSqG14r+2PhPDMAgGg0xNTdHc3Fy26INCgpYglBvXhuerKRQIBGo64S/FGckXTTQzM0M8HicWi3HixAm6urrmCSH+eUmh51klRZVS6wgt51krEcxCNXFrjubaDMuymJmZKVhnpxqUKrT47/lEIkEymfQEc6111v3jpqIGg0GOHz/OWWedlWUXluMjlOrv+O2ImqsP50+5DQQCxONxpqenATxhaDGCTq0W5YXGoVik0PXARVrrGaXUOjJpWzu11k+V6+Ja61cssF8Df1yu6wnZ1KrYdTWve/T2jxF/co+3PfuHb8cqMDENh8OrRkWXlUehXigk/miteeqpp7JEn4GBAQC6u7vzpom5Exw3JdYVhbTWrFu3jgMHDnD2vf8Gc5MzpRRpR2Mqx7v34/E455133rJEnIUErVIQQUlYDK6QeerUqZrVFCo2toWckdwUt3g8Tjwe9/72DcMoKIRorRkeHi74PFuqqLKQkON35Dpfeeu8/e7YlvOslTogQq3IlzJWD/PDUoQW9553o4YtyyKRSADQ3Nyc9/6xLMuzm/7vuRwfoRR/59SP7sva7vvQv+S978PhsGcX3QL7ixV0at35WqhviolCCa31DIDW+pRSan85BSGhfljuqvhSV
```

Comment:
Quick questions (commenting here as I can't on the rendered version of the notebook):
- why are the x labels in the `moran_facette` skewed to the right? The y labels look centered and OK though.
- I'd suggest to move `moran_facette` to `moran_facet` as it's better aligned with the terminology used elsewhere.